Corgibytes.Freshli.Cli/Functionality/LatestOnly.cs

```diff
@@ -0,0 +1,7 @@
+namespace Corgibytes.Freshli.Cli.Functionality;
+
+public enum LatestOnly
+{
+    ShowLatestOnly,
+    WalkBackInRevisionHistory
+}
```

Comment:
I feel like these names could be improved upon. What do you think of:
```suggestion
public enum RevisionHistoryMode
{
    OnlyLatestRevision,
    AllRevisions
}
```

Reply:
After writing that, I think that it's an improvement, and I suspect we could still do better than that. :)

Reply:
Ah yes! Much better, thank you.

Reply:
I've used Rider to do the rename though :D